### ISSUE TITLE
Release notes for Clang and Clang-Tidy v19

### ISSUE THREAD

**Issue by silverqx** [llvm/llvm-project#108354]
Where is the release notes for upcoming v19? For clang, clang-tidy. I have a problem to find the release notes for the upcoming version every time, there are release notes for all previous versions [here](https://releases.llvm.org/) also for the next [v20](https://clang.llvm.org/docs/ReleaseNotes.html) and [here](https://clang.llvm.org/extra/clang-tidy/index.html) but no release notes for the next version v19.

**Comment by silverqx** [comment]
The only release notes I have found is to access it directly on [github](https://github.com/llvm/llvm-project/blob/llvmorg-19.1.0-rc4/clang-tools-extra/docs/ReleaseNotes.rst#id7). But links don't work there of course, is it somewhere live? I would like to prepare my project for v19, especially Clang Tidy checks and this isn't ideal.

**Comment by HerrCai0907** [comment]
in download page. https://releases.llvm.org/download.html

Actually I think it is a little bit counter-intuitive.

**Comment by silverqx** [comment]
But there is no link to v19, only to all older releases <=v18 and to v20 (current git).

**Comment by silverqx** [comment]
I have searched ~1hour and didn't find any, now I'm using release notes in [github tree](https://github.com/llvm/llvm-project/blob/llvmorg-19.1.0-rc4/clang-tools-extra/docs/ReleaseNotes.rst#id7) from v19.1-rc4 tag with the combination of [v20 live pages](https://clang.llvm.org/extra/clang-tidy/checks/list.html) as links don't work.

**Comment by HerrCai0907** [comment]
Maybe you can wait for v19 final release. In this [discuss](https://discourse.llvm.org/t/llvm-19-1-0-final-update-and-possible-rc4/81010) we can found the final release 19.1.0 will be provided in Sept. 17th.

**Comment by HerrCai0907** [comment]
After final release, I think v19 release note will be added in download page.

**Comment by DeepChirp** [comment]
> After final release, I think v19 release note will be added in download page.

But it seems that v19 release note havn't been added until now.

**Comment by EugeneZelenko** [comment]
See https://releases.llvm.org/: https://releases.llvm.org/19.1.0/tools/clang/docs/ReleaseNotes.html and https://releases.llvm.org/19.1.0/tools/clang/tools/extra/docs/ReleaseNotes.html links are there.

**Comment by DeepChirp** [comment]
> See https://releases.llvm.org/: https://releases.llvm.org/19.1.0/tools/clang/docs/ReleaseNotes.html and https://releases.llvm.org/19.1.0/tools/clang/tools/extra/docs/ReleaseNotes.html links are there.

Yes, I meant to say that in the download page (<https://releases.llvm.org/download.html>), the latest version seems to still be 18.1.8.

**Comment by nicovank** [comment]
Looks like it's missing from https://github.com/llvm/www-releases / downloads.html.
@tru is that something you can fix?

To be honest, that downloads page feels redundant.
Maybe just redirect to `index.html` which has almost indentical information?
